core/whistle_number_manager-1.0.0/src/wnm_util.erl

```diff
@@ -432,6 +432,10 @@ maybe_convert_to_e164([Regex|Regexs], Converters, Number) ->
             <<Prefix/binary, Root/binary, Suffix/binary>>
     end.
 
+-spec to_e164_without_plus(ne_binary()) -> ne_binary().
+to_e164_without_plus(Number) ->
+    re:replace(to_e164(Number), "[\+]", "", [{'return', 'binary'}]).
```

Comment:
I've chosen this way because it takes less strings :)
Please let me know if it is need to be redone in pattern matching way and I'll change it